### PR TITLE
[1LP][RFR] Fixes#0

### PR DESCRIPTION
**Pull request by gshefer**
{{pytest:  cfme/tests/containers/test_topology.py  -v --use-provider cm-env2 }}
General test fixes.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit e8540424feb6c7edf5d493cb584ba7282bc4e06e:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/containers/test_provider_configuration_menu.py [hunk 1]
```diff
@@ -52,4 +53,7 @@ def test_remove_selected_containers_provider():
     name = select_first_provider_and_get_its_name()
     ContainersProvider(name).delete(cancel=False)
     sel.refresh()
-    assert not Quadicon(name).exists
+    assert wait_for(lambda *args: not Quadicon(name).exists,
+                    'Waiting for containers provider to be removed.',
+                    delay=5,
+                    num_sec=60.0), 'Failed to remove containers provider'
```
Comment (mfalesni) [inline cfme/tests/containers/test_provider_configuration_menu.py]: missing fail_func that would call selenium's refresh or the reload button click.

File: cfme/tests/containers/test_provider_configuration_menu.py [hunk 2]
```diff
@@ -52,4 +53,7 @@ def test_remove_selected_containers_provider():
     name = select_first_provider_and_get_its_name()
     ContainersProvider(name).delete(cancel=False)
     sel.refresh()
-    assert not Quadicon(name).exists
+    assert wait_for(lambda *args: not Quadicon(name).exists,
```
Comment (mfalesni) [inline cfme/tests/containers/test_provider_configuration_menu.py]: *args is unnecessary here

**Comment by dajoRH** [comment]
Lint report for commit 9c5a30ae2eec92d7891f538e68168cbf37426411:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2ccd98b40e518f324ee11a79d63e159110a26c3e:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6055b7151c710e11f7ec5b14b491de591f590f19:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 18af2d9d35a7f30afbc0ae680489cc50d9f3f0cc:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0ce47252158c5d2cfcfa9a1eab0ae6ceace84f48:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f58ea2361b2fd1611d272007bb28cf95759123ee:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit dad92ece5b3dfb7259afb0f29ab3f965feb4ece9:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 231e0d661610aa5bd6e3218f48621a4b3951502a:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 003bedc3c891022d1f568828b52edc88133213bb:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit beb234be245a099d879ac626d79c4b0f47b88fd5:

`cfme/tests/containers/test_topology.py`:
- :red_circle: Line 24:7: **N801** *class names should use CapWords convention*
- :red_circle: Line 41:9: **N806** *variable in function should be lowercase*
- :red_circle: Line 49:9: **N806** *variable in function should be lowercase*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0a54f088af263c1e42621632f953e54e3cb1cca0:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e7436b321223fff6366991b0f770150961eb274c:

`utils/browser.py`:
- :red_circle: Line 295:7: **N801** *class names should use CapWords convention*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ef0e0f7be8963083a19e219b0d39cf14dbb89849:

`utils/browser.py`:
- :red_circle: Line 295:7: **N801** *class names should use CapWords convention*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d65efebabc4ef1006d068f6cfc4cff32888de0eb:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b6ac6e7a6bb8a2ff279354811067bb3be8169e6a:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c8efdb68ea11980bb580d54ed2f711ac572632a0:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2e52ecc0b86e0fc095624fe347187e18024cf525:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**